test/evergreen.yml

```diff
@@ -1493,6 +1493,57 @@ variables:
           format_args: disagg.mode=follower ops.verify=1 runs.mirror=1
           times: 5
 
+  - &format-stress-test-disagg-leader-palite
```

Comment:
The name contains `stress`, is it a stress test or a smoke one? The config suggests one table and 3 minutes (we could be very stressful during that time but just wanted to double check)